### ISSUE TITLE
calc_demand takes the values of the demand_eq

### ISSUE THREAD

**Issue by robfairh** [arfc/d3ploy#281]
I was trying to run this file where a series of initial facilities are deployed and then the enrichment plant and source are deployed with d3ploy.
https://github.com/robfairh/transition-scenarios/blob/experimental6/input/eg01-eg29/eg01-eg29-flatpower-d3ployC-onemixer0-ma2.xml

```
    <institution>
    <config>
    <DemandDrivenDeploymentInst>
        <calc_method>ma</calc_method>
        <demand_eq>60000</demand_eq>
        <facility_commod>
        <item>
          <facility>source</facility>
          <commod>sourceout</commod>
        </item>
        <item>
          <facility>enrichment</facility>
          <commod>enrichmentout</commod>
        </item>
        </facility_commod>
        <facility_capacity>
        <item>
          <facility>source</facility>
          <capacity>1e8</capacity>
        </item>    
        <item>
          <facility>enrichment</facility>
          <capacity>1e100</capacity>
        </item>
        </facility_capacity>
    </DemandDrivenDeploymentInst>
    </config>
    <name>timeseriesinst</name>
    </institution>
```
This is the table of calc_demand for enrichmentout:
![image](https://user-images.githubusercontent.com/43791150/61892566-b041a200-aed1-11e9-9a48-46f6f2d9f156.png)

And this is the table for the demand of enrichmentout:
![image](https://user-images.githubusercontent.com/43791150/61892729-09a9d100-aed2-11e9-836c-86a876e2da8a.png)

**Comment by robfairh** [comment]
This is what it is doing now, if there isn't a demand at that time step, it assigns the value of the demand_eq at t=0. Also this causes that the inital calc_demand for all the commodities is the demand_eq at t=0.
```        
if time not in self.commodity_demand[commod]:
    t = 0
    self.commodity_demand[commod][time] = eval(self.demand_eq)
```

**Comment by robfairh** [comment]
closed by #282
